CHANGES.rst

```diff
@@ -1,11 +1,14 @@
 Changes
 =======
 
-Version 0.9.2 (UNRELEASED)
+Version 0.9.2 (2023-12-01)
 --------------------------
 
-- Changes the pagination logic to avoid counting twice the total amount of records.
-- Fixes job status consumer exception while attempting to fetch workflow engine logs for workflows could not have been successfully scheduled.
+- Adds automated multi-platform container image building for amd64 and arm64 architectures.
+- Changes the workflow deletion endpoint to return a different and more appropriate message when deleting all the runs of a workflow.
+- Changes CVMFS support to allow users to automatically mount any available repository.
+- Changes how pagination is performed in order to avoid counting twice the total number of records.
+- Fixes job status consumer exception while attempting to fetch workflow engine logs for workflows that could not have been successfully scheduled.
```

Comment:
```suggestion
- Fixes job status consumer exception while attempting to fetch workflow engine logs for workflows that were not successfully scheduled.
```
I think the original was a bit hard to read, wdyt about this small change to improve readability?